substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/c/libc/GLibc.java

```diff
@@ -40,12 +40,12 @@ public void prepare(Path directory) {
     }
 
     @Override
-    public List<String> getCCompilerOptions() {
+    public List<String> getAdditionalQueryCodeCompilerOptions() {
         return new ArrayList<>();
```

Comment:
Why not use `Collections.emptyList()` here and also below.
There is no value in allowing these return values to be mutable lists.

Reply:
I agree, changing it to `Collections.emptyList()`.